styles/styles.css

```diff
@@ -82,15 +83,24 @@ nav {
   color: var(--black-color);
 }
 
+::-webkit-scrollbar {
+  width: 4px;
+}
+::-webkit-scrollbar-track {
+  background-color: var(--inactive-dark);
+}
+::-webkit-scrollbar-thumb {
+  background-color: var(--black-color);
+}
+
 /* -----sidebar ends here----- */
 
 .main-content-container {
   z-index: 1;
-    margin-top: 6rem;
-    margin-left: 30rem;
+  margin-top: 6rem;
+  margin-left: 30rem;
   flex-grow: 1;
   padding: 2rem 1rem;
-  /* background-color: cornflowerblue; */
 }
 
 .display-container{
```

Comment:
You can use a prettier code extension if you are not using it already. this will automatically align the code for you.

Reply:
I do use it, maybe I might have not saved the code